package.json

```diff
@@ -61,6 +66,7 @@
     "chalk": "^2.3.0",
     "compression": "^1.6.2",
     "debug": "^3.1.0",
+    "each-async": "^1.0.0",
```

Comment:
Not sure we need each-async as an explicit dependency.

Reply:
Hey i was having issues with it locally for some reason. I think i may have caused it while experimenting with eslint teamcity reporter but not 100% it was caused this way. I will remove it from being explicitly required.